configs/clip/metafile.yml

```diff
@@ -0,0 +1,317 @@
+      URL: https://github.com/open-mmlab/mmclassification/blob/dev-1.x/mmcls/models/backbones/vision_transformer.py
+      Version: v1.0.0
+
+Models:
+  - Name: clip-vit-base-p16_laion2b_3rdparty_ft_in1k-384px
+    Metadata:
+      FLOPs: 49370078208.0
+      Parameters: 86568424
+      Training Data:
+        - ImageNet-1k
+    In Collections: CLIP
+    Results:
+      - Dataset:
+        - ImageNet-1k
+        Metrics:
+          Top 1 Accuracy: 86.52
+          Top 5 Accuracy: 97.97
+        Task: Image Classification
+    Weights: /mnt/petrelfs/share_data/lirongjie/clip_backbone_ckpt/vit_base_patch16_clip_384.laion2b_ft_in1k.bin
+    Config: configs/clip/vit-base-p16_pt-64xb64_in1k-384.py
+    Converted From:
+      Weights: https://huggingface.co/timm/vit_base_patch16_clip_384.laion2b_ft_in1k
+      Code: https://github.com/rwightman/pytorch-image-models
+
+  - Name: clip-vit-base-p32_laion2b_3rdparty_ft_in1k-224px
+    Metadata:
+      FLOPs: 4364335104.0
+      Parameters: 88225000
+      Training Data:
+        - ImageNet-1k
+    In Collections: CLIP
+    Results:
+      - Dataset:
+        - ImageNet-1k
+        Metrics:
+          Top 1 Accuracy: 82.46
+          Top 5 Accuracy: 96.12
+        Task: Image Classification
+    Weights: /mnt/petrelfs/share_data/lirongjie/clip_backbone_ckpt/vit_base_patch32_clip_224.laion2b_ft_in1k.bin
+    Config: configs/clip/vit-base-p32_pt-64xb64_in1k-224.py
+    Converted From:
+      Weights: https://huggingface.co/timm/vit_base_patch32_clip_224.laion2b_ft_in1k
+      Code: https://github.com/rwightman/pytorch-image-models
+
+  - Name: clip-vit-base-p16_openai_3rdparty_ft_in12k_in1k-384px
+    Metadata:
+      FLOPs: 49370078208.0
+      Parameters: 86568424
+      Training Data:
+        - ImageNet-1k
+        - ImageNet-21k
+    In Collections: CLIP
+    Results:
+      - Dataset:
+        - ImageNet-1k
+        - ImageNet-21k
+        Metrics:
+          Top 1 Accuracy: 86.87
+          Top 5 Accuracy: 98.05
+        Task: Image Classification
+    Weights: /mnt/petrelfs/share_data/lirongjie/clip_backbone_ckpt/vit_base_patch16_clip_384.openai_ft_in12k_in1k.bin
+    Config: configs/clip/vit-base-p16_pt-64xb64_in1k-384.py
+    Converted From:
+      Weights: https://huggingface.co/timm/vit_base_patch16_clip_384.openai_ft_in12k_in1k
+      Code: https://github.com/rwightman/pytorch-image-models
+
+  - Name: clip-vit-base-p16_openai_3rdparty_ft_in12k_in1k-224px
+    Metadata:
+      FLOPs: 16855600128.0
+      Parameters: 86568424
+      Training Data:
+        - ImageNet-1k
+        - ImageNet-21k
+    In Collections: CLIP
+    Results:
+      - Dataset:
+        - ImageNet-1k
+        - ImageNet-21k
+        Metrics:
+          Top 1 Accuracy: 85.99
+          Top 5 Accuracy: 97.72
+        Task: Image Classification
+    Weights: /mnt/petrelfs/share_data/lirongjie/clip_backbone_ckpt/vit_base_patch16_clip_224.openai_ft_in12k_in1k.bin
+    Config: configs/clip/vit-base-p16_pt-64xb64_in1k-224.py
```

Comment:
The config name should be `vit-base-p16_64xb64_in1k.py`. You don't need `pt` to specify a config for pre-train or fine-tuning by now.
And if the image size is not 224px, please use name like `vit-base-p16_64xb64_in1k-384px.py`